src/getComponents.js

```diff
@@ -20,16 +39,34 @@ const getThemePath = theme => {
 const getPluginPath = getPath('plugins');
 const getLanguagePath = getPath('languages');
 
-export default ({ languages = [], plugins = [], theme, css = false } = {}) => [
-    ...getLoader(config, [...languages, ...plugins]).getIds().reduce((deps, dep) => {
-        // Plugins can have language dependencies.
-        const add = [isPlugin(dep) ? getPluginPath(dep) : getLanguagePath(dep)];
+/**
+ * @param {Options} [options]
+ * @returns {string[]}
+ *
+ * @typedef Options
+ * @property {string[] | 'all'} [languages]
+ * @property {string[]} [plugins]
+ * @property {string} [theme]
+ * @property {boolean} [css]
+ */
+export default (options = {}) => {
+    let { languages = [], plugins = [], theme, css = false } = options;
```

Comment:
```suggestion
export default ({ languages = [], plugins = [], theme, css = false } = {}) => {
```

Reply:
I wanted to say that I can't add types via JSDoc like this, but apparently, [you can](https://stackoverflow.com/a/42898969/7595472). Will change.